lib/hyrax/resource_sync/change_list_writer.rb

```diff
@@ -0,0 +1,90 @@
+module Hyrax
+  module ResourceSync
+    # TODO: the big assumption I'm making here is that the repository has fewer
+    # than 50,000 resources to list. The Sitemap protocol is limited at 50,000
+    # items, so if we require more than that, we must have multiple Change
+    # lists and add a Change List Index to point to all of them.
+    class ChangeListWriter
```

Comment:
We absolutely need too, but each page can have 50k objects. See https://github.com/projecthydra-labs/hyrax/issues/59 if you'd like to work on that.